eng/pipelines/common/platform-matrix.yml

```diff
@@ -158,6 +158,30 @@ jobs:
         crossBuild: true
         ${{ insert }}: ${{ parameters.jobParameters }}
 
+# Linux Bionic arm
+
+- ${{ if containsValue(parameters.platforms, 'linux_bionic_arm') }}:
+  - template: xplat-setup.yml
+    parameters:
+      jobTemplate: ${{ parameters.jobTemplate }}
+      helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
+      variables: ${{ parameters.variables }}
+      osGroup: linux
+      osSubgroup: _bionic
+      archType: arm
+      targetRid: linux-bionic-arm
+      platform: linux_bionic_arm
+      shouldContinueOnError: ${{ parameters.shouldContinueOnError }}
+      container: linux_bionic
+      jobParameters:
+        runtimeFlavor: mono
```

Comment:
Is this correct?

Reply:
It matches the definition for arm64. It quite possibly doesn't do anything. (I already tried an official build with this and it did produce a good looking package)